packages/components/psammead-brand/src/index.jsx

```diff
@@ -73,24 +73,40 @@ const BrandSvg = styled.svg`
   }
 `;
 
-const Brand = ({ brandName, svg }) => (
+/* eslint-disable react/prop-types */
+const StyledBrand = ({ svg, brandName }) => (
+  <Fragment>
+    {svg && (
+      <StyledSpan>
+        <BrandSvg
+          viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
+          xmlns="http://www.w3.org/2000/svg"
+          focusable="false"
+          aria-hidden="true"
+          ratio={svg.ratio}
+        >
+          {svg.group}
+        </BrandSvg>
+        <VisuallyHiddenText>{brandName}</VisuallyHiddenText>
+      </StyledSpan>
+    )}
+  </Fragment>
+);
+
+/* eslint-disable react/prop-types */
+const BrandWithLink = ({ link, brandName, svg }) => (
+  <StyledLink href={link}>
+    <StyledBrand svg={svg} brandName={brandName} />
+  </StyledLink>
+);
+
+const Brand = ({ brandName, svg, link }) => (
   <StyledWrapper>
     <ConstraintWrapper>
-      {svg && (
-        <StyledLink href="https://www.bbc.co.uk/news">
-          <StyledSpan>
-            <BrandSvg
-              viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
-              xmlns="http://www.w3.org/2000/svg"
-              focusable="false"
-              aria-hidden="true"
-              ratio={svg.ratio}
-            >
-              {svg.group}
-            </BrandSvg>
-            <VisuallyHiddenText>{brandName}</VisuallyHiddenText>
-          </StyledSpan>
-        </StyledLink>
+      {link ? (
```

Comment:
Why does this conditional exist if `link` should always exist?

Reply:
According to this comment. If the link is not provided in any case we should just display the branding that doesn't link to anywhere.
https://github.com/bbc/psammead/issues/500#issuecomment-492656690